main.scss

```diff
@@ -84,9 +93,44 @@
 	.o-typography-list--unordered {
 		@include oTypographyListUnordered;
 	}
+	.o-typography-footer {
```

Comment:
We should probably start thinking about adding an `if silent` clause in each file and add these there to organise it a bit better, this is getting a bit too long